content/operate/rc/subscriptions/_index.md

```diff
@@ -26,13 +26,12 @@ Here's a quick comparison of each plan:
 | Feature | Redis Cloud Essentials (free) | Redis Cloud Essentials (paid) | Redis Cloud Pro |
 |:-----|:-------:|:----:|:-----:|
 | Memory size | 30 MB | 250 MB-12 GB | 50 TB |
-| Concurrent connections | 30 | 256-Unlimited | Unlimited |
-| Security | role-based auth<br/>password protection<br/>encryption in transit | role-based auth<br/>password protection<br/>SSL & SIP auth<br/>encryption in transit | role-based auth<br/>password protection<br/>SSL & SAIP auth<br/>encryption in transit<br/>encryption at rest |
-| Admin REST API | No | No | Yes |  
-| Support | Basic | Standard | Hourly: Enhanced<br/>Annual: Premium |
+| Concurrent connections | 30 | 256-10000 | Unlimited |
+| Security | role-based auth<br/>password protection<br/>encryption in transit | role-based auth<br/>password protection<br/>TLS auth<br/>encryption in transit | role-based auth<br/>password protection<br/>TLS auth<br/>encryption in transit<br/>encryption at rest |
```

Comment:
Might be worth adding capitals to the start of each item here. This is in line with "Selected additional features" below and helps clarify what the separate items are (eg, "password protection" wraps in the third column on my browser and reads as if it were two separate items).
```suggestion
| Security | Role-based auth<br/>Password protection<br/>Encryption in transit | Role-based auth<br/>Password protection<br/>TLS auth<br/>Encryption in transit | Role-based auth<br/>Password protection<br/>TLS auth<br/>Encryption in transit<br/>Encryption at rest |
```